README.md

```diff
@@ -95,15 +95,15 @@ jobs:
   advanced-ci:
     runs-on: [ubuntu-20.04]
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
 
       - name: Test My Solution
         continue-on-error: true
         run: dotnet test ./src/my-solution.sln --logger "trx" --configuration Release --results-directory ../../test-results
       
       - name: Process trx reports
         id: process-trx
-        uses: im-open/process-dotnet-test-results@v2.2.0
+        uses: im-open/process-dotnet-test-results@v2.2.1
```

Comment:
This will be 2.2.2 now right?

Reply:
Oh man I thought I was being clever...but this one won't have a minor increment.  Shoot!

Reply:
The updated versions are pushed.  Sorry about that confusion.  I'm confused. :)